lib/plugins/globalid.js

```diff
@@ -9,6 +9,9 @@ var noop = require('../utils/misc').noop
 var cookie = require('../vendor/js-cookies')
 
 function cacheSuccess (result, cookieName) {
+  if(!result['global_id']) {
```

Comment:
Tests are failing because of style issue. You'll need a space between `if` and `(!result['global_id'])`
```suggestion
  if (!result['global_id']) {
```

Reply:
@tung-vu-td  fixed the style issue. could you check that?